File: prime.py
num = 11

if num > 1:
	
	for i in range(2, int(num/2)+1):
		
		if (num % i) == 0:
			print(num, "is not a prime number")
			break
	else:
		print(num, "is a prime number")
else:
	print(num, "is not a prime number")

File: scripts/prime.java
import java.util.Scanner;

public class prime {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
     System.out.print("Input a number: ");
     int n = in.nextInt(); 
     in.close();
     int flag = 0;
     if (n<0){
        System.out.println("invalid value ");
        System.exit(0);
     }
     if (n ==  0 || n == 1){
    flag = 1;
     }
        for (int i = 2 ; i < n/2; i++){
            if (n % i == 0){
                flag= 1;
            }          
        }
        if (flag > 0 ){
            System.out.println("  non prime ");          
        }
        else {
            System.out.println(" prime");
        }    }    
}
